core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -73,6 +88,26 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
+    $scope.openExploration = function(status, explorationId) {
+      if (status === 'private') {
+        $scope.showExplorationEditor(explorationId);
+      } else {
+        DashboardBackendApiService.fetchExplorationStats(explorationId).then(
+          function(response) {
+            $scope.explorationStats[explorationId] = response.data;
+          }, function(errorResponse) {
+            if (FATAL_ERROR_CODES.indexOf(errorResponse.status) !== -1) {
```

Comment:
@AllanYangZhou This seems be to a subset of https://github.com/oppia/oppia/issues/2142 . To address this issue here, I've added a constant to base.js and used that as a warning everywhere when the request fails without these error codes! (See https://github.com/oppia/oppia/pull/2296/commits/c4b7e7233e10b07d797a51fc04fa2ee0170bac20)